synapse/logging/opentracing.py

```diff
@@ -641,57 +641,16 @@ def extract_text_map(carrier):
 # Tracing decorators
 
 
-def trace(func):
+def trace(func=None, operation_name=None):
```

Comment:
`operation_name` is a bit unweildy. `opname`?

Reply:
operation_name is the same variable used in openracing. I think it's clearer if we stick to their naming scheme.